taca/storage/storage.py

```diff
@@ -231,9 +231,9 @@ def _send_to_swestore(f, dest, remove=True):
         """
         if not filesystem.is_in_swestore(f):
             logger.info("Sending {} to swestore".format(f))
-            misc.call_external_command('iput -K -P {file} {dest}'.format(file=f, dest=dest),
-                    with_log_files=True)
-            logger.info('Run {} sent correctly and checksum was okay.'.format(f))
+            misc.call_external_command('iput -P {file} {dest}'.format(file=f, dest=dest),
+                    with_log_files=True, prefix=f.replace('.tar.bz2',''), log_dir="swestore_logs")
+            logger.info('Run {} sent to swestore.'.format(f))
```

Comment:
Yes, that is why we added `prefix` while calling the external command, so it will create a log file for called command (`iput` in this case) as {run_id}_iput.err instead of just `iput.err`. So it will not be overwritten :+1: 

And the `logger` instance will append the logs to the main log file (`~/log/taca.log`)